DataFormats/Headers/CMakeLists.txt

```diff
@@ -25,6 +25,7 @@ O2_GENERATE_LIBRARY()
 set(TEST_SRCS
   test/testDataHeader.cxx
   test/testTimeStamp.cxx
+  test/test_HeartbeatFrame.cxx
```

Comment:
the name does not look  consistent with the other tests

Reply:
a naming scheme will only be imposed in #546 and uses the `test_` prefix. But then the prefix is automatically added. In that respect the old names are not consistent. I expect this to be changed anyhow when #546 is merged.